modules/drivers/exasol/src/metabase/driver/exasol.clj

```diff
@@ -147,5 +149,20 @@
              (dissoc details :host :port :dbname :db :ssl))
       (sql-jdbc.common/handle-additional-options details)))
 
+(defmethod sql-jdbc.conn/data-warehouse-connection-pool-properties :exasol
```

Comment:
Wondering if it okay for us to change the code each time we need to rebalance this

I was thinking to pass id to driver 

properties          (data-warehouse-connection-pool-properties driver id)

and then use it as 

(or (config/config-int :mb-jdbc-data-warehouse-max-connection-pool-size-#id) 
(config/config-int :mb-jdbc-data-warehouse-max-connection-pool-size)
                                      15)

But actually dunno if it possible in clojure 🤔

Reply:
That way we could update the pool distribution without changing the code

Reply:
yeah I am also a little concerned about hard-coding pools

Reply:
The code is not hard-coded.
`config/config-int :helios-connection-pool-size` is to get the value from `HELIOS_CONNECTION_POOL_SIZE` Environment variable, if the var doesn't exist then it is default to 20.

In case we need to rebalance connection distribution, we just need to update the environment variables: `HELIOS_CONNECTION_POOL_SIZE`, `HELIOS_PLUS_CONNECTION_POOL_SIZE`, and `HELIOS_FINANCE_CONNECTION_POOL_SIZE`, no need to change the code.

Reply:
I was only thinking about the possibility that some of those users (metabase_infosec, metabase_legal, metabase_processing, metabase_regulatory_complience, metabase_risk) will not be satisfied with 5 connections, and we would have to update the code for them.

If it's not the case, I'm fine with the current code.

Reply:
Good point, updated the code to set default Helios connections to 10

Reply:
Actually, I will update the code to accept connection pool as an argument when configuring Exasol connection in Metabase admin page.

Reply:
Yeah, I know it's not hard coded in terms of the size, but hard coded pool names (meaning only 3 pools are configurable at all).
+1 to an input box on the page if this is not too difficult

Reply:
Implemented a more elegant way to configure Helios connection pool on admin page. Default is set to 10 connections.

Reply:
In case of connection redistribution, we just need to update connection pools on admin page.